Source/JavaScriptCore/API/JSRemoteInspector.h

```diff
@@ -50,10 +50,8 @@ JS_EXPORT void JSRemoteInspectorSetLogToSystemConsole(bool) JSC_API_AVAILABLE(ma
 JS_EXPORT bool JSRemoteInspectorGetInspectionEnabledByDefault(void) JSC_API_AVAILABLE(macos(10.11), ios(9.0));
 JS_EXPORT void JSRemoteInspectorSetInspectionEnabledByDefault(bool) JSC_API_DEPRECATED("Use JSGlobalContextSetInspectable on a single JSGlobalContextRef.", macos(10.11, JSC_MAC_TBA), ios(9.0, JSC_IOS_TBA));
 
-// FIXME: <rdar://104354248> SPI annotated with TBA availability information is not usable in WebCore/WebKit on
-// downlevel builds. These SPI are available starting in JSC_MAC_TBA and JSC_IOS_TBA.
-JS_EXPORT bool JSRemoteInspectorGetInspectionFollowsInternalPolicies(void);
-JS_EXPORT void JSRemoteInspectorSetInspectionFollowsInternalPolicies(bool);
+JS_EXPORT bool JSRemoteInspectorGetInspectionFollowsInternalPolicies(void) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
Is TBA still accurate for these now?

Reply:
It can be the versions we seeded to developers yesterday. But also fine if that is being done as a followup for all the applicable uses. Landing with the TBAs would be a way to ensure the fix works across our builders, though!

Reply:
That's my plan. I'll land with TBAs, and set this to the correct version as part of https://bugs.webkit.org/show_bug.cgi?id=252437 (Update WebKit TBA versions for macOS 13.3 and iOS 16.4)